scripts/setup_postgres_azurerole.ps1

```diff
@@ -1,10 +1,10 @@
-$POSTGRES_HOST = ((azd env get-values | Select-String -Pattern "POSTGRES_HOST") -replace '^POSTGRES_HOST=', '')
-$POSTGRES_USERNAME = ((azd env get-values | Select-String -Pattern "POSTGRES_USERNAME") -replace '^POSTGRES_USERNAME=', '')
-$APP_IDENTITY_NAME = ((azd env get-values | Select-String -Pattern "SERVICE_WEB_IDENTITY_NAME") -replace '^SERVICE_WEB_IDENTITY_NAME=', '')
+$POSTGRES_HOST = (azd env get-value POSTGRES_HOST)
```

Comment:
Recommend trying this out in error scenarios to see if it looks right. It may print out more error spew: `'POSTGRES_HOST' is not found in environment values` and also the additional errors after

Reply:
Ah you mean, try it with a non-existing env key? Or a repo that has no azd?

Reply:
Try it without running `azd up`

Reply:
Yes, you're right, error looks like--

```
@pamelafox ➜ /workspace (azdenvgetval) $ ./scripts/setup_postgres_database.sh 
Creating Python virtual environment in .venv...
Installing dependencies from "requirements.txt" into virtual environment (in quiet mode)...
usage: setup_postgres_database.py [-h] [--host HOST] [--username USERNAME] [--password PASSWORD] [--database DATABASE]
                                  [--sslmode SSLMODE]
setup_postgres_database.py: error: unrecognized arguments: ensuring environment exists: environment not specified ensuring environment exists: environment not specified ensuring environment exists: environment not specified
```

I think I need a check at the beginning to see if an azd environment exists yet.

Reply:
I've added a check for exit code of 1 for the first command. Thanks!